data/human/human missions.txt

```diff
@@ -3996,3 +3996,91 @@ mission "Fumbling Mugger"
 event "fumbling mugger followup timer"
 
 
+
+mission "Dangerous Games 1"
+	name "Dragonriders"
+	description "A lively man named Karengo and a few passionate youth are traveling with you to see the dragons on <stopover>. Take the group to <destination> and return the group back to <source> once they've finished."
```

Comment:
Should this say 'youths'?

Also, as far as I can tell, the source and destination for this mission will be the same?

Reply:
I tried to create a situation where you would pick them up at the origin planet, drop them off at Skymoot, and then fly them back to the origin planet. I used stopover for this, but I might not have used it correctly.

Reply:
Yes, this mission should have you go to Skymoot and then back to where you started. `<destination>` is the final destination, which is in this case the source as well. So saying to take them to the destination and then back the the source makes no sense. You want to tell the player to take them to the stopovers and then to the destination/source.

Reply:
I tried fixing it, is it ok now?

Reply:
You can mark this as resolved.